cmd/state/internal/cmdtree/shim.go

```diff
@@ -0,0 +1,43 @@
+package cmdtree
+
+import (
+	"github.com/ActiveState/cli/internal/captain"
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/primer"
+	"github.com/ActiveState/cli/internal/runners/shim"
+)
+
+func newShimCommand(prime *primer.Values) *captain.Command {
+	runner := shim.New(prime)
+
+	params := shim.Params{}
+	cmd := captain.NewCommand(
+		"shim",
+		locale.T("shim_description"),
+		[]*captain.Flag{
+			{
+				Name:        locale.T("flag_state_shim_language"),
+				Description: locale.T("flag_state_shim_language_description"),
+				Value:       &params.Language,
+			},
+		},
+		[]*captain.Argument{
+			{
+				Name:        locale.T("args_state_shim_script"),
+				Description: locale.T("args_state_shim_script_description"),
+				Required:    true,
+				Value:       &params.Script,
+			},
+		},
+		func(ccmd *captain.Command, args []string) error {
+			if len(args) > 0 {
+				args = args[1:]
+			}
+			return runner.Run(params, args...)
+		},
+	)
+	cmd.SetSkipUpdate(true)
```

Comment:
Should set this on the update command as well so we can get rid of http://github.com/ActiveState/cli/blob/201911db318226fdbbb9c81cfced9dcb321690d9/cmd/state/autoupdate.go#L23-L23